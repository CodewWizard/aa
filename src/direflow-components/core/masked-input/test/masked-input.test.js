import React from "react";
import renderer from "react-test-renderer";
import MaskedInput from "../masked-input";

describe("MaskedInput", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<MaskedInput options={{ phone: true, phoneRegionCode: "US" }} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
