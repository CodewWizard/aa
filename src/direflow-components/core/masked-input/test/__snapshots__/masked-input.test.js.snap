// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MaskedInput renders correctly 1`] = `
<input
  onBlur={[Function]}
  onChange={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  type="text"
  value=""
/>
`;
